pipelines/optimus/README.md

```diff
@@ -44,5 +234,16 @@ Following are the the types of files produced from the pipeline.
 | zarr_output_files | {unique_id}.zarr!.zattrs | | zarr store? sparse matrix? | | Yes | | 
 | loom_output_file | output.loom | Loom | Loom | Loom file with expression data and metadata | N/A | N/A |
 
-### Components of Optimus
-The source code is available from [Github](https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/optimus/Optimus.wdl), an overview of the pipeline can be found on the [HCA Data Portal](https://prod.data.humancellatlas.org/) and the benchmarking that was performed on the pipeline can be found [here](https://docs.google.com/document/d/158ba_xQM9AYyu8VcLWsIvSoEYps6PQhgddTr9H0BFmY/edit#heading=h.calfpviouwbg). Some of the tasks in Optimus use the [sctools](https://github.com/HumanCellAtlas/sctools) library of utilities for large scale distributed single cell data processing, and [Picard](https://broadinstitute.github.io/picard/) tools, a set of command line tools for manipulating high-throughput sequencing data in formats such as SAM/BAM/CRAM and VCF.
+# Versioning
+
+| Optimus Release Version | Date | Release Note | 
+| :---: | :---: | :---: |
+| v1.3.6 (current) | 09/23/2019 | Optimus now optionally outputs a Loom formatted count matrix, with the default being true. |
+| v1.3.3 | 08/29/2019 | This version and newer have been validated to additionally support Mouse data. The gene expression per cell is now counted by gencode geneID instead of gene name. There is an additional output mapping geneID to gene name provided. This is a breaking change. | 
+| v1.0.0 |03/30/2019 | Initial pipeline release. Validated on hg38 gencodev27. | 
+
+# Additional Notes
+
+Some of the tasks in Optimus use the [sctools](https://github.com/HumanCellAtlas/sctools) library of utilities for large scale distributed single cell data processing, and [Picard](https://broadinstitute.github.io/picard/) tools, a set of command line tools for manipulating high-throughput sequencing data in formats such as SAM/BAM/CRAM and VCF.
```

Comment:
I would rather that this section's information is put in where relevant throughout the documentation.. seems weird to note it as an afterthought? 

And can we add contact information somewhere? Top or bottom, either way. 

Maybe a header like 

| Pipeline Version | Date Updated | Documentation Author | Questions or Feedback |
| :----: | :---: | :----: | :--------------: |
| optimus_v1.3.6 | October 10, 2019 | [Elizabeth Kiernan](mailto:ekiernan@broadinstitute.org) | Please file github issues in skylab or contact [Kylee Degatano](mailto:kdegatano@broadinstitute.org) |